lib/rules/function-calc-no-unspaced-operator/index.mjs

```diff
@@ -242,20 +251,80 @@ function insertCharAtIndex(str, index, char) {
 }
 
 /**
- * @param {import('postcss-value-parser').Node | undefined} node
- * @returns {node is import('postcss-value-parser').SpaceNode}
+ * @param {import('@csstools/css-parser-algorithms').TokenNode} node
+ * @returns {{value: string, startIndex: number, endIndex: number} | undefined}
+ */
+function getDelimToken(node) {
+	const [type, value, startIndex, endIndex] = node.value;
+
+	if (type !== 'delim-token') return;
+
+	return { value, startIndex, endIndex };
+}
+
+/**
+ * @param {import('@csstools/css-parser-algorithms').ComponentValue} node
+ * @param {Set<string>} operators
+ * @returns {boolean}
+ */
+function isOperatorNode(node, operators) {
+	if (!isTokenNode(node)) return false;
+
+	return operators.has(getDelimToken(node)?.value ?? '');
+}
+
+/**
+ * @param {import('@csstools/css-parser-algorithms').WhitespaceNode} node
+ * @returns {string}
  */
-function isSingleSpace(node) {
-	return node != null && isSpace(node) && node.value === ' ';
+function getWhitespace(node) {
+	return node.value[0]?.[1] ?? '';
 }
 
+/** @see https://drafts.csswg.org/css-values/#typedef-calc-value */
+const OPERAND_TOKEN_TYPES = new Set([
+	'number-token',
+	'dimension-token',
+	'percentage-token',
+	'ident-token',
+]);
+
 /**
- * @param {import('postcss-value-parser').Node | undefined} node
- * @param {Set<string>} [operators]
- * @returns {node is import('postcss-value-parser').WordNode}
+ * @param {import('@csstools/css-tokenizer').CSSToken} token
+ * @returns {boolean}
  */
-function isOperator(node, operators = OPERATORS) {
-	return node != null && isWord(node) && operators.has(node.value);
+function isOperandToken(token) {
+	const [type, value] = token;
+
+	if (!OPERAND_TOKEN_TYPES.has(type)) return false;
+
+	if (type === 'ident-token' && !calcKeywords.has(value)) return false;
+
+	return true;
+}
+
+/**
+ * @param {import('@csstools/css-parser-algorithms').ComponentValue | undefined} node
+ * @returns {boolean}
+ */
+function isOperandNode(node) {
+	return isTokenNode(node) && isOperandToken(node.value);
+}
```

Comment:
Aren't these also valid operands?

```
calc((10px) + 5px)
calc(calc(10px) + 5px)
```

Reply:
You have added examples of such cases in your other PR.
Let's add one test case here.
If it fails add `skip: true` for now.